scene/main/node.cpp

```diff
@@ -1409,7 +1409,9 @@ void Node::add_sibling(Node *p_sibling, bool p_force_readable_name) {
 
```

Comment:
`ERR_FAIL_COND_MSG(data.blocked > 0`
should more likely be
`ERR_FAIL_COND_MSG(data.parent->data.blocked > 0`

Reply:
Testing out this fix right now, thank you

Reply:
I would also reorder checks a bit, two involving `p_sibling` together and two involving `data.parent` together.

Reply:
I wonder if we should add a `data.blocked` around the actual child additions in `_add_child_nocheck`, we do for removing

Reply:
The name `nocheck` suggests that it shouldn't do any checking :P

Reply:
Don't mean checking, but `data.block++` :P